app/src/main/java/org/thoughtcrime/securesms/conversation/v2/ConversationActivityV2.kt

```diff
@@ -1815,10 +1862,32 @@ class ConversationActivityV2 : PassphraseRequiredActionBarActivity(), InputBarDe
     }
 
     override fun sendVoiceMessage() {
+        // When the record voice message button is released we always need to reset the UI and cancel
+        // any further recording operation..
         hideVoiceMessageUI()
         window.clearFlags(WindowManager.LayoutParams.FLAG_KEEP_SCREEN_ON)
         val future = audioRecorder.stopRecording()
         stopAudioHandler.removeCallbacks(stopVoiceMessageRecordingTask)
+
+        // ..but we'll bail without sending the voice message & inform the user that they need to press and HOLD
+        // the record voice message button if their message was less than 1 second long.
+        val inputBar = binding?.inputBar
```

Comment:
All these nullable binding creating a lot of `!!` usage is pretty ugle and makes the code so much worse.
We will soon phase out view binding when we use Compose, but could we at least change the declaration to something like this, so that the rest of the code gets much cleaner?
```
private var _binding: ActivityConversationV2Binding? = null
private val binding get() = _binding!!
```

Reply:
Done